beacon-chain/blockchain/core_test.go

```diff
@@ -192,20 +203,21 @@ func TestCanProcessBlock(t *testing.T) {
 		t.Fatalf("CanProcessBlocks failed: %v", err)
 	}
 	if !canProcess {
-		t.Errorf("Should be able to process block, could not")
+		t.Error("Should be able to process block, could not")
 	}
 
 	// Attempting to try a block with that fails the timestamp validity
 	// condition.
 	block = types.NewBlock(1000000)
+	block.InsertParentHash(parentHash)
```

Comment:
I would just do `block = types.NewBlock(&pb.BeaconBlockResponse{ParentHash: parentHash, SlotNumber: slotNumber})` throughout these tests and remove the `InsertParentHash` function definition. Its a little more verbose, but I don't like the idea of exposing a new public method just to facilitate testing.